src/analysis/excluded_terms_in_mondo/excluded_terms_in_mondo.py

```diff
@@ -0,0 +1,196 @@
+"""Excluded terms xrefed-in Mondo
+
+Resources
+- GitHub issue: https://github.com/monarch-initiative/mondo-ingest/issues/31
+- GitHub PR: https://github.com/monarch-initiative/mondo-ingest/pull/35
+"""
+import os
+import sys
+from argparse import ArgumentParser
+from typing import Dict, List
+
+import pandas as pd
+
+
+# Vars
+# # Static
+THIS_SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
+ANALYSIS_DIR = os.path.join(THIS_SCRIPT_DIR, '..')
+PROJECT_DIR = os.path.join(ANALYSIS_DIR, '..', '..')
+MAPPINGS_DIR = os.path.join(THIS_SCRIPT_DIR, 'ignored', 'mappings')
+REPORTS_DIR = os.path.join(PROJECT_DIR, 'src', 'ontology', 'reports')
+SIGNATURE_FILES_DIR = REPORTS_DIR
+OUTDIR = REPORTS_DIR
+# # Config
+CONFIG = {
+    # todo: get list from mondo config yml:
+    'ontologies': ['doid', 'icd10cm', 'icd10who', 'ncit', 'omim', 'ordo'],
```

Comment:
Yep! I can't remember if we discussed but this is on my radar.

Reply:
If this were a parameter to the script, then I would do what we talked about elsewhere and move these into a test.

However, this is not currently a param to the script:
```make
$(REPORTDIR)/excluded_terms_in_mondo_xrefs.tsv $(REPORTDIR)/excluded_terms_in_mondo.tsv: $(TMPDIR)/mondo.sssom.tsv
	python3 ../analysis/excluded_terms_in_mondo.py \
	--mondo-mappings-path $(TMPDIR)/mondo.sssom.tsv \
	--outpath $@
```

The way that we've designed this so far has been to run all ontologies at once. If you want, we can change this to be like a lot of our other `make` goals: Running one ontology at a time, and generating single output files for each ontology.

If we do that, do you think you could pitch in with your unix-foo to create a make goal that combines all of these ontology files? I'm guessing you'd prefer this combination step to be done in `make` rather than in a Python script. I can take a stab if you want me to spend the time on it.

Reply:
Yeah, I think we should change it. This is just going to fail every time unless things are pre-downloaded.

Also one of the artefact dependencies I need are `mirror/%.owl` files. I'm having the issue again where I have a couple `.ttl` files (ICD10CM and ICD10WHO) that I need an `.owl` version for. Have to look up how I did that last time.

Reply:
We decided I'll change it to make per ontology. Nico will write something that combines them.